[2;32m2017-04-28 10:59:03,761 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://118.190.90.1:3306/fastfun?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-04-28 10:59:05,590 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-04-28 10:59:05,591 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-04-28 10:59:05,591 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-04-28 10:59:06,486 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-04-28 10:59:06,543 - INFO  [main:SelectorManager@37] - Creating 4 reactors...
[m[2;35m2017-04-28 10:59:06,700 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-04-28 10:59:06,770 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 118.190.90.1:11211
[m[2;32m2017-04-28 10:59:06,788 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2017-04-28 10:59:06,812 - INFO  [main:ServerBootstrap@129] - Application PID:	8172
[m[2;35m2017-04-28 10:59:26,302 - WARN  [Thread-33:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :321, action:com.klw.fastfun.pay.common.action.CommonAction, method:getOtherPayCodes
[m[2;35m2017-04-28 11:00:22,373 - WARN  [Thread-13:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :90, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2017-04-28 11:00:47,813 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x5e4f414c, /127.0.0.1:51503 :> /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-04-28 11:00:47,824 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xe326b845, /127.0.0.1:51505 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-04-28 11:00:47,831 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x399b09ee, /127.0.0.1:51504 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
